src/webui/www/private/views/filters.html

```diff
@@ -281,6 +317,11 @@
             toggleCategoryDisplay(filterItem.id);
         });
 
+        const clearStatusFilter = () => { setStatusFilter("all"); };
+        const clearCategoryFilter = () => { setCategoryFilter(CATEGORIES_ALL); };
+        const clearTagFilter = () => { setTagFilter(TAGS_ALL); };
+        const clearTrackerFilter = () => { setTrackerFilter(TRACKERS_ALL); };
+
```

Comment:
Seems they are local to the function. Please move them into the function (just above the `switch` statement).

Reply:
I've refactored existing code to use these new functions.